isb_web/metrics.py

```diff
@@ -2,15 +2,14 @@
 import time
 from typing import Optional
 
-from fastapi import FastAPI
+from fastapi import APIRouter
 from starlette.responses import PlainTextResponse
 
 from isb_web.isb_solr_query import solr_counts_by_authority
 from isb_web.sqlmodel_database import SQLModelDAO, things_by_authority_count_dict
 
-metrics_api = FastAPI()
+router = APIRouter(prefix="/metrics")
```

Comment:
This is the preferred way (according to FastAPI docs) to mount subpaths underneath an app.

Reply:
Yep, this looks good